presto-main/src/main/java/com/facebook/presto/type/BooleanOperators.java

```diff
@@ -100,6 +100,13 @@ public static double castToDouble(@SqlType(StandardTypes.BOOLEAN) boolean value)
     }
 
     @ScalarOperator(CAST)
+    @SqlType(StandardTypes.REAL)
+    public static long castToReal(@SqlType(StandardTypes.BOOLEAN) boolean value)
+    {
+        return value ? 1065353216 : 0; // 1065353216 means 1.0f
```

Comment:
Sorry now I see why you are adding CAST BOOLEAN to REAL in the previous commits. You can directly discuss with me if you feel I misunderstand anything :). 

Let's do this:
1. Make this commit before the CAST JSON TO REAL COMMIT
2. Add separate test cases for CAST BOOLEAN to REAL

Sorry about the confusion!

For the code, what about this? :

```
return value ? floatToRawIntBits(1.0) : floatToRawIntBits (0.0);
```